components/prospects/ProfileSettings.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import IconButton from '@mui/material/IconButton';
+import Router from 'next/router';
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from '@mui/material';
```

Comment:
Please make sure you don't have any lines with more than 80 characters. If you do, continue to the next line.